client/layout/masterbar/style.scss

```diff
@@ -919,7 +913,7 @@ body.is-mobile-app-view {
 
 	.masterbar__item-subitems {
 		min-width: 264px !important;
-		max-height: 94px;
```

Comment:
And this reverts a change I made. :-)

Reply:
Unintented change, I had the file open before git pull :man_facepalming: